src/Stratis.Bitcoin.Features.BlockStore/BlockStoreCache.cs

```diff
@@ -133,22 +94,14 @@ public void AddToCache(Block block)
             uint256 blockid = block.GetHash();
             this.logger.LogTrace("({0}:'{1}')", nameof(block), blockid);
 
-            if (!this.cache.TryGetValue(blockid, out Block existingBlock))
-                this.cache.Set(blockid, block, this.blockEntryOptions);
-
-            this.logger.LogTrace("(-)[{0}]", existingBlock != null ? "ALREADY_IN_CACHE" : "ADDED_TO_CACHE");
+            this.logger.LogTrace("(-)");
```

Comment:
I believe `this.logger.LogTrace("(-)");` should be the very last line in the method ;)

Reply:
+1

Reply:
Fixed